src/openrct2-ui/windows/GameBottomToolbar.cpp

```diff
@@ -480,6 +480,7 @@ class GameBottomToolbar final : public Window
                 break;
             case WIDX_PARK_RATING:
                 ft.Add<int16_t>(gameState.ParkRating);
+                ft.Add<money64>(gameState.ParkValue);
```

Comment:
Should we hide the Park Value when hovering over the finances tooltip as it provides two ways to show the same information?